### PR TITLE
Optimalization inference

### PR DESCRIPTION
**Pull request by VidunderGunder**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,3 +1,8 @@
 {
-  "python.analysis.extraPaths": ["AdaBins", "yolov5", "TACO"]
+  "python.analysis.extraPaths": [
+    "AdaBins",
+    "yolov5",
+    "TACO"
+  ],
+  "python.pythonPath": "/usr/local/anaconda3/envs/njord/bin/python"
```
Comment (VidunderGunder) [inline .vscode/settings.json:7]: This is a private setting that will only work on very specific installations, no?

File: README.md [hunk 2]
```diff
@@ -88,7 +88,7 @@ conda create -n vision python=3.6.6 anaconda
 Install PyTorch (tested on 1.7.0 and 1.7.1)
 
 ```shell
-conda install pytorch torchvision torchaudio cudatoolkit=10.2 -c pytorch
+conda install pytorch torchvision cudatoolkit=10.2 -c pytorch
```
Comment (VidunderGunder) [inline README.md:91]: Makes sense. Good catch 👌

**Comment by VidunderGunder** [comment]
@Uzaaft I can't test the performance gains by tweaking the precision until tomorrow or even thursday. Hopefully it makes a significant difference 🤞🤞

**Comment by Uzaaft** [comment]
I deleted the branch,  so you can just go ahead and close this PR.

**Pull request closed without merging**